src/app/modules/submission/multirespondent-submission/multirespondent-submission.utils.ts

```diff
@@ -42,19 +42,17 @@ export const createMultirespondentSubmissionDto = (
 }
 
 export const retrieveWorkflowStepEmailAddresses = (
-  formWorkflow: FormWorkflowDto,
-  nextWorkflowStep: number,
+  step: FormWorkflowStepDto,
   responses: FieldResponsesV3,
-): Result<string[], InvalidWorkflowTypeError> => {
```

Comment:
I would recommend retaining `neverthrow` typings  since it can be extremely difficult to debug if the controller was not handling the correct types or the function returning the wrong types.